src/Psalm/Internal/CallMap.php

```diff
@@ -12374,19 +12381,19 @@
 'sqlite_unbuffered_query' => ['SQLiteUnbuffered', 'dbhandle'=>'', 'query'=>'string', 'result_type='=>'int', 'error_msg='=>'string'],
 'sqlite_valid' => ['bool', 'result'=>''],
 'SQLiteDatabase::__construct' => ['void', 'filename'=>'', 'mode='=>'int|mixed', '&error_message'=>''],
-'SQLiteDatabase::arrayQuery' => ['array', 'query'=>'string', 'result_type='=>'int', 'decode_binary='=>'bool'],
-'SQLiteDatabase::busyTimeout' => ['int', 'milliseconds'=>'int'],
-'SQLiteDatabase::changes' => ['int'],
-'SQLiteDatabase::createAggregate' => ['', 'function_name'=>'string', 'step_func'=>'callable', 'finalize_func'=>'callable', 'num_args='=>'int'],
-'SQLiteDatabase::createFunction' => ['', 'function_name'=>'string', 'callback'=>'callable', 'num_args='=>'int'],
-'SQLiteDatabase::exec' => ['bool', 'query'=>'string', 'error_msg='=>'string'],
-'SQLiteDatabase::fetchColumnTypes' => ['array', 'table_name'=>'string', 'result_type='=>'int'],
-'SQLiteDatabase::lastError' => ['int'],
-'SQLiteDatabase::lastInsertRowid' => ['int'],
-'SQLiteDatabase::query' => ['SQLiteResult', 'query'=>'string', 'result_type='=>'int', 'error_msg='=>'string'],
+'SQLiteDatabase::arrayQuery' => ['array', 'dbhandle'=>'', 'query'=>'string', 'result_type='=>'int', 'decode_binary='=>'bool'],
```

Comment:
`arrayQuery` as a _method_ does not accept dbhandle (at least according to PHP documentation). That's probably true for other methods here as well.

Reply:
Yeah, I'm going through and fixing that now, one sec